plugins/balloontoolbar/plugin.js

```diff
@@ -169,9 +170,24 @@
 	};
 
 	/**
-	 * Hides the toolbar and removes it from the DOM.
+	 * Cleares Balloon Toolbar from each menu items.
+	 *
+	 * @since 4.9.0
+	 */
+	CKEDITOR.ui.balloonToolbar.prototype.clearItems = function() {
```

Comment:
I'm wondering if this method should be public. I don't see any real use-cases for it except of destroying the toolbar.